docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: global
+title: Running Apache Hive with Alluxio
+nickname: Apache Hive
+group: Frameworks
+priority: 2
+---
+
+This guide describes how to run [Apache Hive](http://hive.apache.org/) with Alluxio, so
+that you can easily store Hive tables into Alluxio at various storage level.
+
+# Prerequisites
+
+The prerequisite for this part is that you have
+[Java](Java-Setup.html). Alluxio cluster should also be
+set up in accordance to these guides for either [Local Mode](Running-Alluxio-Locally.html) or
+[Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+
+Please [Download Hive](http://hive.apache.org/downloads.html).
+
+# Configuration
+
+Apache Hive allows you to use Alluxio through a generic file system wrapper for the Hadoop file system.
+Therefore, the configuration of Alluxio is done mostly in Hive and its under computing frameworks.
+
+#### Set property in `hive-site.xml`
+
+You need to add the following property to `hive-site.xml` in your Hive installation `conf` directory
+
+```xml
+<property>
+   <name>fs.defaultFS</name>
+   <value>alluxio://<master_hostname>:19998</value>
+</property>
+```
+
+#### Set the rest properties in Apache Hadoop MapReduce
+
+If you run Hive on Apache Hadoop MapReduce, Hive can read configirations from Hadoop configuration files. In addition,
+Hive's Hadoop jobs will store its immediate results on Alluxio.
+
+You need to add the following properties to `core-site.xml` file in Hadoop:
+
+```xml
+<property>
+   <name>fs.alluxio.impl</name>
+   <value>alluxio.hadoop.FileSystem</value>
+</property>
+<property>
+   <name>fs.AbstractFileSystem.alluxio.impl</name>
+   <value>alluxio.hadoop.AlluxioFileSystem</value>
+</property>
+```
+
+modify `$HADOOP_CLASSPATH` by changing `hadoop-env.sh` to:
+
+```bash
+$ export HADOOP_CLASSPATH=/<PATH_TO_ALLUXIO>/core/client/target/alluxio-core-client-{{site.ALLUXIO_RELEASED_VERSION}}-jar-with-dependencies.jar:${HADOOP_CLASSPATH}
+```
+
+Distributing the Alluxio Client Jar in Hadoop MapReduce:
+
+- You must place the client jar
+`alluxio-core-client-{{site.ALLUXIO_RELEASED_VERSION}}-jar-with-dependencies.jar`
+(located in the `/<PATH_TO_ALLUXIO>/core/client/target/` directory), in the `$HADOOP_HOME/lib`
+(may be `$HADOOP_HOME/share/hadoop/common/lib` for different versions of Hadoop) directory of every
+MapReduce node, and restart all of the NodeManagers.
+
+# Distribute the Alluxio Client jar
```

Comment:
how is this section different from the previous section about putting the jar in `$HADOOP_HOME/lib`?